scripts/add-validation-to-crds/parse-crds.go

```diff
@@ -131,6 +131,10 @@ func processFile(ctx context.Context, p string) error {
 }
 
 func addRefsToCRD(crd *apiextensions.CustomResourceDefinition) error {
+	if crd.Spec.Names.Kind == "IAMAuditConfig" {
+		// no refs here
```

Comment:
Non-blocker: We'll want to provide consistent interface eventually so maybe add a TODO here?